spec/springboard/client/resource_spec.rb

```diff
@@ -134,6 +140,37 @@
     end
   end
 
+  describe "count" do
+    let(:response_data) {
+      {
+        :status => 200,
+        :body => {total: 123}.to_json
+      }
+    }
+
+    it "should call the client's count method with the resource's URI" do
+      expect(client).to receive('count').with(resource.uri)
+      resource.__send__('count')
```

Comment:
There's no need to use `__send__` with a literal method name. Just call the method instead.

Reply:
All set